introduction.md

```diff
@@ -39,3 +39,6 @@ Ruby
 Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?